magenta/reviews/assets/pixelrnn_make_masks.py

```diff
@@ -1,3 +1,11 @@
+# Author: Kyle Kastner
```

Comment:
No worries. I will keep you all posted on the status.

On Mon, Jul 11, 2016 at 3:21 PM, cinjon <notifications@github.com> wrote:

> In magenta/reviews/assets/pixelrnn_make_masks.py
> <https://github.com/tensorflow/magenta/pull/89#discussion_r70348107>:
>
> > @@ -1,3 +1,11 @@
> > +# Author: Kyle Kastner
>
> I was slightly wrong on this. Fjord is looking into it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tensorflow/magenta/pull/89/files/d7f24547bdf3e466bfba02abf17f4970894a58d8#r70348107>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABfbHYBkODx7PcawWmUZCWsxhalNXyE-ks5qUsHsgaJpZM4JJ0JG>
> .
>